api/src/main/java/eu/coatrack/api/Proxy.java

```diff
@@ -55,6 +56,10 @@ public Proxy(String id) {
 
     private String configServerPassword;
 
+    private LocalDateTime lastCallTimeToAdmin;
```

Comment:
The name should make clear that this is the admin's perspective (so it is not the time the proxy tried to call but the time admin received a call). Maybe:
```suggestion
    private LocalDateTime timeOfLastSuccessfulCallToAdmin;
```

Reply:
I agree